src/main/java/de/cyface/collector/MainVerticle.java

```diff
@@ -107,10 +112,14 @@ private Router setupRoutes() {
 	}
 
 	private void startHttpServer(final Router router, final Future<Void> startFuture) {
-		String certificateFile = this.getClass().getResource("/localhost.jks").getFile();
-		HttpServerOptions options = new HttpServerOptions().setSsl(true).setKeyStoreOptions(new JksOptions().setPath(certificateFile).setPassword("secret"));
-		
-		vertx.createHttpServer(options).requestHandler(router::accept).listen(config().getInteger("http.port", 8080),
+		final int httpPort = Parameter.HTTP_PORT.intValue(vertx, 8080);
+		final String defaultCertificateFile = this.getClass().getResource("/localhost.jks").getFile();
+		final String certificateFile = Parameter.TLS_KEYSTORE.stringValue(vertx, defaultCertificateFile);
+
+		final HttpServerOptions options = new HttpServerOptions().setSsl(true)
+				.setKeyStoreOptions(new JksOptions().setPath(certificateFile).setPassword("secret"));
```

Comment:
same as above: the pw should be some constant and documented if this is something to care about or an open TODO